src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
```

Comment:
This is a code quality issue. It's generally not recommended to set a global font-weight as it can lead to unexpected results. It would be better to specify the font-weight individually for each element as needed.